dbms/src/Storages/Page/V3/CheckpointFile/Proto/data_file.proto

```diff
@@ -0,0 +1,53 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.#pragma once
+
+syntax = "proto3";
+
+package DB.PS.V3.CheckpointProto;
+
+import "common.proto";
+
+// This prefix is only for debugging purpose.
```

Comment:
Why "only for debugging purpose"? It is used in many places.

Reply:
The prefix of the data file is not used. In the read path we should just read data from the data file using a specified seek offset.

Reply:
OK. Can you put more comments here about the information?